corehq/apps/reports/standard/project_health.py

```diff
@@ -81,6 +81,8 @@ def __init__(self, domain, month, users, has_filter, performance_threshold, prev
             total_users_by_month=0,
             percent_active=0,
             performing=num_performing_user,
+            delta_num_high_performing_users=0,
+            delta_num_low_performing_users=0,
```

Comment:
Any reason these two aren't just passed in to the constructor for this class as well?